src/clients/BundleDataClient/utils/SuperstructUtils.ts

```diff
@@ -123,7 +124,7 @@ const nestedV3BundleFillsSS = record(
 );
 
 export const BundleDataSS = object({
-  bundleBlockRanges: array(array(number())),
```

Comment:
I just don't know whether we want to impose a type here if we don't use the property. Ideally I'd like to make this property optional, I think I can use the `type()` [superstruct type](https://docs.superstructjs.org/api-reference/types#type) instead of `object`.